app/views/courses/show.html.erb

```diff
@@ -1,17 +1,50 @@
+<script>
+function drawFlashcard() {
+  var xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    console.log(this.status);
+    if (this.readyState == 4 && this.status == 200) {
+     populateFlashcard(this.responseText)
+    }
+  };
+  xhttp.open("GET", '<%= url_for(controller: 'courses', action: 'flashcard') %>', true);
+  xhttp.send();
+}
+
+function populateFlashcard(json) {
+  var student = JSON.parse(json);
+  document.getElementById("student-name").innerHTML = student.name;
+  document.getElementById("student-info").innerHTML = student.email;
+}
+</script>
 <center>
   <br>
     <div>
-      <%= link_to "Edit Course", edit_user_course_path(@user, @course) %>
-      <%= link_to "All Courses", user_courses_path(@user) %>
-      <%= link_to "Account", user_path(@user) %>
-      <%= link_to "Add Student", new_user_course_student_path(@user, @course) %>
-      <h1><%= @course.title %></h1>
+      <h1> <%= @course.code %>: <%= @course.title %></h1>
       <hr>
       <div class = "container">
         <div class="row">
-          <div class="col-lg-7 border-right">
-          <h2>Students</h2>
-          <table class="experiment-table">
+          <div class="col-lg-4">
+            <h3> Actions</h3>
+            <%= link_to "Edit Course", edit_user_course_path(@user, @course) %><br>
+            <%= link_to "All Courses", user_courses_path(@user) %><br>
+            <%= link_to "Account", user_path(@user) %><br>
+            <%= link_to "Add Student", new_user_course_student_path(@user, @course) %><br>
+          </div>
+          <div class = "col-lg-4"></div>
+          <div class = "col-lg-4">
+            <h3>Flashcards</h3>
+            <div id="flashcard-info">
+              <h4 id="student-name">Name</h4>
+              <p id="student-info">info</p>
```

Comment:
Capitalize I in info?

Reply:
That part is just temporary, we will probably pre-draw a student in the future